docs/guides/autoscaler-diskless-topics.md

```diff
@@ -0,0 +1,221 @@
+---
+page_title: "Autoscale service capacity based on CPU usage"
+---
+
+# Autoscale service capacity based on CPU usage
+
+~> **Limited availability**
+Autoscaling Inkless clusters is in the limited availability stage. To enable this feature, contact the [Aiven support team](mailto:support@aiven.io).
+
+Autoscaling automatically adjusts your service capacity based on CPU usage. It helps maintain strong performance during traffic spikes by scaling up resources when the CPU load is consistently high. It also controls costs by scaling down when demand decreases. Autoscaling operates within the minimum and maximum plans that you define.
+
+## Prerequisites
+
+Autoscaling is only available for Inkless services. Inkless services are `aiven_kafka` services with the following:
+  -  A plan with the `-inkless` suffix
+  -  Tiered storage enabled
```

Comment:
The `diskless` topics should be also on, `kafka_diskless.enabled`. This is a service level feature, and if I recall correctly, config cannot be set on afterwards easily.